### PR TITLE
Get rid of vendor/. Use peep. Rejoice.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -97,30 +97,76 @@ Tests
...
+Here's what that folder contains:
 
-There are a small number of compiled packages, including the MySQL Python
-client. You can install these using ``pip`` or via a package manager.
-To use ``pip``, you only need to do the following.
+- ``compiled.txt`` - contains dependencies that require a compiler and may
+  need to be treated differently dependending on environment
+
+- ``default.txt`` - contains the default dependencies that are used during
+  in all environments
```
Comment (robhudson) [inline docs/development.rst]: We can drop "during".

File: manage.py [hunk 2]
```diff
@@ -1,26 +1,7 @@
...
-site.addsitedir(path('vendor'))
-
-# Move the new items to the front of sys.path.
-new_sys_path = []
-for item in list(sys.path):
-    if item not in prev_sys_path:
-        new_sys_path.append(item)
-        sys.path.remove(item)
-sys.path[:0] = new_sys_path
-
```
Comment (robhudson) [inline manage.py]: No more path hacking! :clap: :fireworks: :tada:

File: requirements/default.txt [hunk 3]
```diff
@@ -1,119 +1,79 @@
...
-
+django-flat-theme==1.1.1
 django-honeypot==0.5.0
-
 django-jinja==1.4.1
-
 django-memcached-hashring==0.1.2
-
+django-mozilla-product-details==0.7.1
 django-nose==1.4.2
```
Comment (robhudson) [inline requirements/default.txt]: Can `django-nose` be moved to dev requirements?

File: requirements/default.txt [hunk 4]
```diff
@@ -1,119 +1,79 @@
...
-
+django-babel==0.4.0
+django-cacheback==0.9.1
 django-celery==3.1.17
-
+django-celery-transactions==0.3.6
 django-constance==1.1.1
-
+django-dbgettext==0.1.1
 django-extensions==1.5.9
```
Comment (robhudson) [inline requirements/default.txt]: dev requirement?

File: requirements/default.txt [hunk 5]
```diff
@@ -1,119 +1,79 @@
...
+funcsigs==0.4
+futures==3.0.3
+hash-ring==1.3.1
+html5lib==0.999999
+importlib==1.0.3
+jsonpickle==0.9.2
+kombu==3.0.29
+logilab-astng==0.24.3
+logilab-common==1.1.0
+mock==1.3.0
```
Comment (robhudson) [inline requirements/default.txt]: We learned that mock is required for production. :cry:

File: requirements/default.txt [hunk 6]
```diff
@@ -1,119 +1,79 @@
...
+nose==1.3.7
```
Comment (robhudson) [inline requirements/default.txt]: dev requirement?

File: requirements/default.txt [hunk 7]
```diff
@@ -1,119 +1,79 @@
...
+oauthlib==1.0.3
+ordereddict==1.1
+pbr==1.8.1
+polib==1.0.7
+puente==0.4.1
+Pygments==2.0.2
+pyquery==1.2.9
```
Comment (robhudson) [inline requirements/default.txt]: Is pyquery a dev requirement?

File: requirements/dev.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
+coverage==3.7.1
+django-debug-toolbar==1.4
+django-devserver==0.8.0
+gunicorn==19.4.1
```
Comment (robhudson) [inline requirements/dev.txt]: Do we actually use this?

**Comment by jezdez** [comment]
something is wrong with travis right now :( closing..

File: requirements/default.txt (hunk 3 above)
Comment (jezdez) [inline requirements/default.txt]: I think so.

File: requirements/default.txt (hunk 4 above)
Comment (jezdez) [inline requirements/default.txt]: No, we use a few model fields form that package.

File: requirements/dev.txt (hunk 8 above)
Comment (jezdez) [inline requirements/dev.txt]: Yeah we recently started to use it in development to be able to use whitenoise and have a dev/prod parity for serving static files.

File: requirements/default.txt (hunk 7 above)
Comment (jezdez) [inline requirements/default.txt]: No, it's a "prod" requirement.

File: requirements/default.txt (hunk 5 above)
Comment (jezdez) [inline requirements/default.txt]: :crying_cat_face:

File: manage.py (hunk 2 above)
Comment (jezdez) [inline manage.py]: Party!!

File: docs/development.rst (hunk 1 above)
Comment (jezdez) [inline docs/development.rst]: Good catch :)

**Pull request closed without merging**